v3/pkg/application/linux_cgo.go

```diff
@@ -1010,17 +1010,41 @@ func onDragNDrop(target unsafe.Pointer, context *C.GdkDragContext, x C.gint, y C
 
 //export onKeyPressEvent
 func onKeyPressEvent(widget *C.GtkWidget, event *C.GdkEventKey, userData unsafe.Pointer) C.gboolean {
-	windowId := uint(*((*C.uint)(userData)))
-	fmt.Println("onKeyPressEvent", windowId)
-	/*
-		windowKeyEvents <- &windowKeyEvent{
+    accelerator := getKeyboardState(event)
```

Comment:
Please update `getKeyboardState` to return a `(string, bool)` and check if it was successfully converted before submitting it to the channel.
I think a warning log message would be nice if something doesn't match up to the keycodes instead of sending it on the channel.